infra/common/src/test/java/cn/hippo4j/common/executor/ThreadFactoryBuilderTest

```diff
@@ -0,0 +1,45 @@
+package cn.hippo4j.common.design.builder;
+
+import cn.hippo4j.common.design.builder.ThreadFactoryBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.concurrent.ThreadFactory;
+import java.lang.Thread.UncaughtExceptionHandler;
+
+public class ThreadFactoryBuilderTest {
```

Comment:
Methods should be covered by multiple unit test cases instead of one unit test covering all methods.

Reply:
ok

Reply:
https://github.com/opengoofy/hippo4j/pull/1340